src/rgw/rgw_op.h

```diff
@@ -353,6 +353,21 @@ class RGWGetObj : public RGWOp {
 
   int get_data_cb(bufferlist& bl, off_t ofs, off_t len);
 
+  int submit_l2_request(string dest, off_t obj_ofs, size_t len, off_t read_ofs, void* args, size_t (*write_cb)(void *, size_t, size_t, void *)) {
+    return s->submit_http_req(dest, obj_ofs, len, read_ofs, args, write_cb);
+  }
```

Comment:
is this a placeholder? i don't see why this calls through `req_state::submit_http_req()` when all that does is `return 0;`

Reply:
yes, will move the L2 functionality to a separate PR